plugins/utilities.json

```diff
@@ -3,6 +3,101 @@
   "description": "Utilities",
   "plugins_base_url": "https://github.com/bombsquad-community/plugin-manager/{content_type}/{tag}/plugins/utilities",
   "plugins": {
+    "sandbox": {
+      "description": "The most advanced bombsquad mod ever imo, which can basically do almost everything in the game, literally.",
+      "external_url": "",
+      "authors": [
+        {
+          "name": "BrotherBoard",
+          "email": "thehero2012008@gmail.com",
+          "discord": "BrotherBoard"
+        }
+      ],
+      "versions": {
+        "1.2.0": {
+          "api_version": 8,
+          "commit_sha": "4855584",
+          "released_on": "18-05-2024",
+          "md5sum": "ce6e2acb7dca2d41df1d5b5dcb0064a1"
+        }
+      }
+    },
+    "updown": {
+      "description": "Adds UP and DOWN buttons in party window to recall messages in chat, just like linux terminal lol",
+      "external_url": "",
+      "authors": [
+        {
+          "name": "BrotherBoard",
+          "email": "thehero2012008@gmail.com",
+          "discord": "BrotherBoard"
+        }
+      ],
+      "versions": {
+        "1.0.0": {
+          "api_version": 8,
+          "commit_sha": "4855584",
+          "released_on": "18-05-2024",
+          "md5sum": "7c597ef8cf7966a056eb5b34c01678eb"
+        }
+      }
+    },
+    "rejoin": {
+      "description": "Adds a button in pause menu, which rejoins current server once clicked! if didn't work, just click again and again. if still, then cope coz party is full.",
+      "external_url": "",
+      "authors": [
+        {
+          "name": "BrotherBoard",
+          "email": "thehero2012008@gmail.com",
+          "discord": "BrotherBoard"
+        }
+      ],
+      "versions": {
+        "2.0.0": {
+          "api_version": 8,
+          "commit_sha": "4855584",
+          "released_on": "18-05-2024",
+          "md5sum": "a34da341e7f2055a1f57f5262fa07c79"
+        }
+      }
+    },
+    "topmsg": {
+      "description": "When chat is muted, see new chat messages on top right! (like kill logs)",
+      "external_url": "",
+      "authors": [
+        {
+          "name": "BrotherBoard",
+          "email": "thehero2012008@gmail.com",
+          "discord": "BrotherBoard"
+        }
+      ],
+      "versions": {
+        "1.0.0": {
+          "api_version": 8,
+          "commit_sha": "4855584",
+          "released_on": "18-05-2024",
+          "md5sum": "74a592e8204e81872752a4a6eff6701e"
+        }
+      }
+    },
+    "sorry": {
+      "description": "Send a random sorry to chat, in case u killed your teammate by mistake or something",
```

Comment:
let's try not use short forms ("u" -> "you"), will be nice to keep things consistent with other plugin descriptions.

Reply:
maybe we could also tell here in the description about how someone would be able to random sorry message (tap on party window).

Reply:
done, conversation resolved.